### PR TITLE
We need to send all safe names with all registrations.

### PR DESCRIPTION
**Pull request by biafra23**
Handles #1489 

Changes proposed in this pull request:
- Register all safes for all owners every time.

@gnosis/mobile-devs

### CODE REVIEW

**Comment by biafra23** [comment]
We could probably delete register() and user registerOwners() instead if it wasn't for 

https://github.com/gnosis/safe-android/pull/1510/files#diff-4e03d291fa24b5aeaaab3f85132dc282170830173513eefb356af3b8b436f253R155
`notificationManager.updateNotificationChannelGroupForSafe(safe)`
and `safeRepository.saveSafeMeta(SafeMetaData(safe.address, true))`

File: app/src/main/java/io/gnosis/safe/notifications/NotificationRepository.kt [hunk 1]
```diff
@@ -131,10 +131,13 @@ class NotificationRepository(
         kotlin.runCatching {
```
Comment (elgatovital) [inline app/src/main/java/io/gnosis/safe/notifications/NotificationRepository.kt:131]: registerSafe(safe: Safe) should be renamed to registerSafe

Comment (biafra23) [reply app/src/main/java/io/gnosis/safe/notifications/NotificationRepository.kt:131]: Right now the parameter is needed to to the onSuccess() stuff. And if we do not need the parameter, we can remove registerSafe and call registerOwners instead. And add the two methods from onSuccess for all safes.

Comment (biafra23) [reply app/src/main/java/io/gnosis/safe/notifications/NotificationRepository.kt:131]: Is it safe to call `notificationManager.updateNotificationChannelGroupForSafe(safe)` and `safeRepository.saveSafeMeta(SafeMetaData(safe.address, true))` in `onSuccess()` for all safes everytime?

Comment (biafra23) [reply app/src/main/java/io/gnosis/safe/notifications/NotificationRepository.kt:131]: For updateNotificationChannelGroupForSafe I think it is safe. But I am not sure about saveSafeMeta.

**Comment by biafra23** [comment]
I don't think it is a good idea to pass the list of safe. For example some places that call registerSafes() do not have a safeRepository yet.

**Review by elgatovital — APPROVED** [review]

**Pull request merged**